collectors/all.h

```diff
@@ -82,6 +82,7 @@
 #define NETDATA_CHART_PRIO_MEM_SYSTEM_AVAILABLE       1010
 #define NETDATA_CHART_PRIO_MEM_SYSTEM_COMMITTED       1020
 #define NETDATA_CHART_PRIO_MEM_SYSTEM_PGFAULTS        1030
+#define NETDATA_CHART_PRIO_MEM_SYSTEM_OOM_KILL        1040
```

Comment:
How about to put the chart right after `NETDATA_CHART_PRIO_MEM_SYSTEM_AVAILABLE`? So you don't need to scroll every time you want to check it?

Reply:
Yes, it makes sense. Moved.